src/engine/SCons/SConf.py

```diff
@@ -592,8 +593,30 @@ def TryBuild(self, builder, text = None, extension = ""):
             raise SCons.Errors.UserError('Missing SPAWN construction variable.')
 
         nodesToBeBuilt = []
+        sourcetext = self.env.Value(text)
+        f = "conftest"
+
+        if text is not None:
+            textSig = SCons.Util.MD5signature(sourcetext)
+            textSigCounter = str(_ac_build_counter[textSig])
+            _ac_build_counter[textSig] += 1
+
+            f = "_".join([f, textSig, textSigCounter])
+            textFile = self.confdir.File(f + extension)
+            textFileNode = self.env.SConfSourceBuilder(target=textFile,
+                                                       source=sourcetext)
+            nodesToBeBuilt.extend(textFileNode)
+
+            source = textFile
+            target = textFile.File(f + "SConfActionsContentDummyTarget")
```

Comment:
This is unfortunate, but without it, we don't have a real `Node` to pass as the `target` argument for the `get_contents` call below, and if there are any emitters in play that want to actually examine the targets, they will fall over. I'm open to suggestions on the name or other approaches to ensure that there is a real `Node` available for use as the `target`.